integration_tests/test_lambda_stack_snapshot.json

```diff
@@ -0,0 +1,90 @@
+{
+  "Resources": {
+    "HelloHandlerServiceRole11EF7C63": {
+      "Type": "AWS::IAM::Role",
+      "Properties": {
+        "AssumeRolePolicyDocument": {
+          "Statement": [
+            {
+              "Action": "sts:AssumeRole",
+              "Effect": "Allow",
+              "Principal": {
+                "Service": "lambda.amazonaws.com"
+              }
+            }
+          ],
+          "Version": "2012-10-17"
+        },
+        "ManagedPolicyArns": [
+          {
+            "Fn::Join": [
+              "",
+              [
+                "arn:",
+                {
+                  "Ref": "AWS::Partition"
+                },
+                ":iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
+              ]
+            ]
+          }
+        ]
+      },
+      "Metadata": {
+        "aws:cdk:path": "ExampleDatadogStack/HelloHandler/ServiceRole/Resource"
+      }
+    },
+    "HelloHandler2E4FBA4D": {
+      "Type": "AWS::Lambda::Function",
+      "Properties": {
+        "Code": {
+          "ZipFile": "test"
+        },
+        "Role": {
+          "Fn::GetAtt": [
+            "HelloHandlerServiceRole11EF7C63",
+            "Arn"
+          ]
+        },
+        "Environment": {
+          "Variables": {
+            "DD_LAMBDA_HANDLER": "lambdaFunction.handler",
+            "DD_TRACE_ENABLED": "true",
+            "DD_LOGS_INJECTION": "true",
+            "DD_SERVERLESS_LOGS_ENABLED": "true",
+            "DD_FLUSH_TO_LOG": "false",
+            "DD_SITE": "datadoghq.com",
+            "DD_API_KEY": "1234"
+          }
+        },
+        "Handler": "/opt/nodejs/node_modules/datadog-lambda-js/handler.handler",
+        "Layers": [
+          "arn:aws:lambda:sa-east-1:464622532012:layer:Datadog-Node14-x:XXX",
+          "arn:aws:lambda:sa-east-1:464622532012:layer:Datadog-Extension:XXX"
+        ],
+        "Runtime": "nodejs14.x",
+        "Tags": [
+          {
+            "Key": "dd_cdk_construct",
+            "Value": "v0.4.0"
```

Comment:
I bet this version number will change over time, causing the snapshot tests to fail. Can we filter it out?